core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -1366,6 +1367,26 @@ protected Void visitDropColumn(DropColumn node, Integer context)
             return null;
         }
 
+        @Override
+        protected Void visitTableExecute(TableExecute node, Integer indent)
+        {
+            builder.append("ALTER TABLE ");
```

Comment:
if you ignore indent (since it must be 0 here), add `checkArgument(indent==0,"...")`

same for others, preexistings (separate pr)

Reply:
I think this is true now. But this is not obvious to me that it must always be true.
E.g. `EXPLAIN ALTER ...` does not bump indent. But it could do so.

Reply:
https://github.com/trinodb/trino/pull/9501